src/peer/negotiator.js

```diff
@@ -226,6 +226,12 @@ class Negotiator extends EventEmitter {
     this._isForceUseStreamMethods =
       browserInfo.name === 'chrome' && browserInfo.major <= 64;
 
+    // If client customizes pcConfig, set 'plan-b'.
+    // TODO: When JS-SDK supports for 'unified-plan', this process should be changed.
+    if (pcConfig) {
```

Comment:
Why is this `if` statement needed?
It seems `pcConfig` always exists wherher client passes own `config` or NOT.

Reply:
It is due to test code. 
But this logic has no relations with product code.
So I deleted it and fixed test code.